nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -177,9 +177,16 @@ Costmap2DROS::on_configure(const rclcpp_lifecycle::State & /*state*/)
     layered_costmap_->addPlugin(plugin);
 
     // TODO(mjeronimo): instead of get(), use a shared ptr
-    plugin->initialize(
+    try {
+      plugin->initialize(
       layered_costmap_.get(), plugin_names_[i], tf_buffer_.get(),
```

Comment:
I'm kind of surprised this isn't caught as a linting error, there is no intent on the arguments

Reply:
Fixed this manually